error in @axosoft/nan when building

When building from source, we get:

```
npm error In file included from ../../@axosoft/nan/nan.h:302:
npm error ../../@axosoft/nan/nan_maybe_43_inl.h:210:15: error: no member named 'SetPrototype' in 'v8::Object'; did you mean 'SetPrototypeV2'?
npm error 210 |   return obj->SetPrototype(isolate->GetCurrentContext(), prototype);
npm error       |               ^~~~~~~~~~~~
npm error       |               SetPrototypeV2
npm error /Users/george/Library/Caches/node-gyp/39.2.3/include/node/v8-object.h:464:37: note: 'SetPrototypeV2' declared here
npm error   464 |   V8_WARN_UNUSED_RESULT Maybe<bool> SetPrototypeV2(Local<Context
npm error > context,
```

@zawata please bump also @axosoft/nan - or can the original nan be used now?

--- a/generate/templates/templates/binding.gyp
+++ b/generate/templates/templates/binding.gyp
@@ -56,7 +56,7 @@
       "include_dirs": [
         "vendor/libv8-convert",
         "vendor/libssh2/include",
-        "<!(node -e \"require('@axosoft/nan')\")"
+        "<!(node -e \"require('nan')\")"
       ],
 
       "cflags": [

--- a/package.json
+++ b/package.json
@@ -38,12 +38,12 @@
     "node": ">= 20"
   },
   "dependencies": {
-    "@axosoft/nan": "^2.22.0-gk.1",
     "@mapbox/node-pre-gyp": "^2.0.0",
     "fs-extra": "^7.0.0",
     "got": "^14.4.7",
     "json5": "^2.1.0",
     "lodash": "^4.17.14",
+    "nan": "^2.23.1",
     "node-gyp": "^11.2.0",
     "tar-fs": "^3.0.9"
   },

